home_default/templates/home_default/base.html

```diff
@@ -1,15 +1,18 @@
+
 <!DOCTYPE html>
+{% load static %}
 <html lang="en">
 <head>
-    <meta charset="UTF-8">
-    <meta content="width=device-width, initial-scale=1.0" name="viewport">
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <link rel="stylesheet" type="text/css" href="{% static 'home_default/css/bootstrap.css' %}" />
     <title>SoloConnect</title>
 </head>
 <body>
 <header>
     <a href="{% url 'home_default:home_page' %}">Home</a>
 </header>
-<div>
+<div class = "container">
```

Comment:
I think that there's a special bootstrap class for messages, but we can put that in later.